geowebcache/diskquota/core/src/main/java/org/geowebcache/diskquota/DiskQuotaMonitor.java

```diff
@@ -394,25 +394,21 @@ private LayerCacheInfoBuilder launchCacheInfoGatheringThreads() throws Interrupt
         cacheInfoBuilder = new LayerCacheInfoBuilder(cacheRoot, cleanUpExecutorService,
                 quotaUsageMonitor);
 
-        for (String layerName : tileLayerDispatcher.getLayerNames()) {
-
-            Quota usedQuota = quotaStore.getUsedQuotaByLayerName(layerName);
-            if (usedQuota.getBytes().compareTo(BigInteger.ZERO) > 0) {
-                log.debug("Using saved quota information for layer " + layerName + ": "
-                        + usedQuota.toNiceString());
-            } else {
-                log.debug(layerName + " has no saved used quota information,"
-                        + "traversing layer cache to compute its disk usage.");
-                TileLayer tileLayer;
-                try {
-                    tileLayer = tileLayerDispatcher.getTileLayer(layerName);
-                } catch (GeoWebCacheException e) {
-                    e.printStackTrace();
-                    continue;
-                }
-                cacheInfoBuilder.buildCacheInfo(tileLayer);
-            }
-        }
+        /*
+         * Commented out, will add layers when needed. See method addLayerToQuotaMonitor
+         * 
+         * for (String layerName : tileLayerDispatcher.getLayerNames()) {
+         * 
+         * Quota usedQuota = quotaStore.getUsedQuotaByLayerName(layerName); if
+         * (usedQuota.getBytes().compareTo(BigInteger.ZERO) > 0) { log.debug(
+         * "Using saved quota information for layer " + layerName + ": " +
+         * usedQuota.toNiceString()); } else { log.debug(layerName +
+         * " has no saved used quota information," +
+         * "traversing layer cache to compute its disk usage."); TileLayer tileLayer; try {
+         * tileLayer = tileLayerDispatcher.getTileLayer(layerName); } catch (GeoWebCacheException e)
+         * { e.printStackTrace(); continue; } cacheInfoBuilder.buildCacheInfo(tileLayer); } } <<<
+         */
+
```

Comment:
Please don't leave commented out code unless it's an example.  For an explanation of when something will happen/be used somewhere else in a way that may be confusing, just use a regular comment.  To explain why you made a change, use a Pull Request comment.